docs/design/datacontracts/datacontracts_design.md

```diff
@@ -16,178 +16,86 @@ The physical layout of this data is not defined in this document, but its practi
-
-It is entirely reasonable for an algorithmic contract to have multiple entrypoints which take different inputs. For example imagine a contract which provides information about a `MethodTable`. It may provide the an api to get the `BaseSize` of a `MethodTable`, and an api to get the `DynamicTypeID` of a `MethodTable`. However, while the set of contracts which describe an older version of .NET may provide a means by which the `DynamicTypeID` may be acquired for a `MethodTable`, a newer runtime may not have that concept. In such a case, it is very reasonable to define that the `GetDynamicTypeID` api portion of that contract is defined to simply `throw new NotSupportedException();`
-
-For simplicity, as it can be expected that all developers who work on the .NET runtime understand C# to a fair degree, it is preferred that the algorithms be defined in C#, or at least psuedocode that looks like C#. It is also condsidered entirely permissable to refer to other specifications if the algorithm is a general purpose one which is well defined by the OS or some other body. (For example, it is expected that the unwinding algorithms will be defined by references into either the DWARF spec, or various Windows Unwind specifications.)
-
-For working with data from the target process/other contracts, the following C# interface is intended to be used within the algorithmic descriptions:
-
-Best practice is to either write the algorithm in C# like psuedocode working on top of the [C# style api](contract_csharp_api_design.cs) or by reference to specifications which are not co-developed with the runtime, such as OS/architecture specifications. Within the contract algorithm specification, the intention is that all interesting api work is done by using an instance of the `Target` class.
-
-## Arrangement of contract specifications in the repo
-
-Specs shall be stored in the repo in a set of directories. `docs/design/datacontracts` Each one of them shall be a seperate markdown file named with the name of contract. `docs/design/datacontracts/datalayout/<contract_name>.md` Every version of each contract shall be located in the same file to facilitate understanding how variations between different contracts work.
-
-### Global Value Contracts
-The format of each contract spec shall be
-
-
-```
-# Contract <contract_name>
+The data descriptor is a logical entity that defines the layout of certain types relevant to one or
+more algorithmic contracts, as well as global values known to the target runtime that may be
+relevant to one or more algorithmic contracts.
 
-Insert description of contract, and what its for here.
+More details are provided in the [data descriptor spec](./data_descriptor.md).  We highlight some important aspects below:
 
-## Version <version_number>
+#### Baseline data descriptor identifier
```

Comment:
This should be moved to `data_descriptor.md` where it talks about baseline descriptor. This higher-level doc should only have a brief mention.